test-cli.py

```diff
@@ -998,6 +1061,76 @@ def test(ctx):
     check_contains('/user/.luarocks/share/lua/5.1/?.lua', run_command([cl, 'cat', uuid+'/stdout']))
 
 
+@TestModule.register('competition')
+def test(ctx):
+    """
+    Sanity-check the competition script.
+    """
+    submit_tag = 'submit'
+    eval_tag = 'eval'
+    with temp_instance() as remote:
```

Comment:
To avoid any clashing in the tags namespace, and also it's harder to keep track of what bundles get created by the competition script (esp if there are any bugs) so it's easier to just do it in a temp instance and then nuke the whole thing.